### PR TITLE
chore: fix badger lookup name, add vaults version defs, add deus price

### PR DESCRIPTION
**Pull request by axejintao**

### CODE REVIEW

File: src/chains/config/fantom.config.ts [hunk 1]
```diff
@@ -162,6 +162,7 @@ export const fantomSetts: VaultDefinition[] = [
     state: VaultState.Guarded,
     protocol: Protocol.OxDAO,
     stage: Stage.Staging,
+    version: VaultVersion.v1_5,
```
Comment (NamelsKing) [inline src/chains/config/fantom.config.ts:165]: dat nice

File: src/indexers/indexer.utils.ts [hunk 2]
```diff
@@ -45,7 +45,7 @@ export async function vaultToSnapshot(chain: Chain, vaultDefinition: VaultDefini
     address: vaultDefinition.vaultToken,
     requireRegistry: false,
     state: VaultState.Open,
-    version: VaultVersion.v1,
+    version: vaultDefinition.version ?? VaultVersion.v1,
```
Comment (NamelsKing) [inline src/indexers/indexer.utils.ts:48]: nullish collision is for rendering mostly, do u need empty string in case?

**Review by NamelsKing — APPROVED** [review]
all is nit, lgtm

File: src/indexers/indexer.utils.ts (hunk 2 above)
Comment (axejintao) [reply src/indexers/indexer.utils.ts:48]: what do you suggest then like `vaultDefinition.version || VaultVersion.v1`?

**Review by mitche50 — APPROVED** [review]

**Pull request merged**